pyrakoon/compat.py

```diff
@@ -1083,13 +1083,17 @@ def getClusterId(self):
 
 # Actual client implementation
 class _ArakoonClient(object, client.AbstractClient, client.ClientMixin):
-    def __init__(self, config, timeout=0):
+    def __init__(self, config, timeout=0, noMasterTimeout=0):
         self._config = config
         self.master_id = None
 
         self._lock = threading.RLock()
         self._connections = dict()
         self._timeout = timeout
+        if (isinstance(timeout, (int, float)) and timeout > 0) or timeout is None:
```

Comment:
Who knows, this is the same validation on the initial timeout param for _ArakoonClient